python/cuxfilter/charts/datashader/datashader.py

```diff
@@ -128,13 +13,16 @@ def scatter(
     aggregate_fn="count",
     point_size=1,
     point_shape="circle",
-    pixel_shade_type="eq_hist",
+    pixel_shade_type="linear",
```

Comment:
Why the change from `eq_hist`?

Reply:
So legends are not yet supported in eq_hist, I wanted to make sure they are rendered in the default examples. Or should it be explicitly mentioned in the docs like - `if legends are a requirement, set pixel_shade_type to 'linear'/'log'`?

Reply:
Oh right, the work for eqhist colorbars is still currently ongoing, I forgot 😞 Well that's annoying. 

I think there should be a note, regardless, since it will help set user expectations and be something that can be pointed to in case of questions.

For the examples can we simply be explicit to set `"linear"` for now (to get a colorbar) but leave the default as eq_hist? Suggest this since:

* I think `eq_hist` is the better default, current colorbar limitations aside
* It avoids having to change an API default value (twice)

Reply:
Yep, that makes more sense